### DISCUSSION TITLE
ElastAlert not alerting, Apscheduler run time was missed by

### DISCUSSION THREAD

**Discussion by shortstack** [jertel/elastalert2#326] [Q&A] [answered]
We have a couple hundred rules and a 12 node Elasticsearch 7 cluster. ElastAlert keeps throwing these errors and rarely actually alerts. 

Running ElastAlert 2 on Ubuntu with Praeco in front.

It's set to run every 10 minutes, connectivity is fine. But it misses more often than it hits at this point. 

What other data should I provide? Suggestions? We didn't have this issue with fewer rules.

![Screenshot_20210703-184246_JuiceSSH~2](https://user-images.githubusercontent.com/1244979/124368393-aa60a980-dc2e-11eb-8f05-18a83141935e.jpg)

**Comment by jertel** [comment, marked as answer]
Hello - Please see #187.

**Reply by shortstack** [reply]
i will grab. thank you!
